addons/javelin/functions/fnc_onOpticDraw.sqf

```diff
@@ -31,24 +31,31 @@ params ["_lastRunFrame", "_currentTarget", "_lockStartTime", "_soundNextPlayTime
 // Get shooter info
 private _currentShooter = if (ACE_player call CBA_fnc_canUseWeapon) then {ACE_player} else {vehicle ACE_player};
 private _currentWeapon = currentWeapon _currentShooter;
-private _currentMagazine = currentMagazine _currentShooter;
+
+// check enabled for the weapon explicitly
+private _weaponEnabled = getNumber (configFile >> "CfgWeapons" >> _currentWeapon >> QGVAR(enabled)) == 1;
```

Comment:
Do we really want to stop checking if the attribute is inherited or not? I can understand for `QGVAR(baseEnabled)`, but I'm not sure about the particular change.
Same question applyies to `ace_missileguidance` below.

Reply:
I don't see why we wouldn't, otherwise we are just continuing the retexture issue for other mods. Inheritance is how config works, explicitly not using here was causing issues and confusion

Reply:
> Inheritance is how config works, explicitly not using here was causing issues and confusion

I fully agree, although I am a bit worried that changing it now *might* cause issues.